returns/context/requires_context.py

```diff
@@ -27,6 +27,9 @@
 # Helpers:
 _FirstType = TypeVar('_FirstType')
 
+# Type Aliases:
```

Comment:
Can you please document it with `#:` 
This way it will be visible to sphinx. 

We also need to describe why do we have this type alias.

Reply:
Done!